service/controller/v12/resource/instance/template/main.json

```diff
@@ -62,6 +62,13 @@
       "metadata":{
         "description":"Output value of the worker subnet ID as referenced from the virtual network setup."
       }
+    },
+    "zones":{
+      "type":"array",
+      "defaultValue": [1],
```

Comment:
Does this mean that by default we would use multi-AZ and then set it to use a single zone (zone 1) by default?
If so, wouldn't we want to completely skip multi-AZ parameters if the customer didn't specify them (so we keep existing/old behavior)?

Reply:
For me it's ok to have default value in template, but we should ensure that even with single zone request, we randomize the tenant cluster specific AZ on creation. This brings little bit more resilience for customer so that not every single tenant cluster goes down when single AZ goes down on specific region.

Reply:
I don't think we can randomize it. If we allow customers to select specific AZ's and they choose one single AZ, it would be a surprise to them to find the cluster using a different AZ because it was randomized it.

Reply:
Sorry about mixing up different things. This here is fine. On principal operator must not change a value set in CR. In some cases if user did not specify AZ, then it's usually randomly selected in `api` (i.e. before writing that into CR), but in case the end user always picks up an AZ, then there's no randomization at all. Nevertheless, for me this is fine.